### PR TITLE
allow custom region url

### PR DESCRIPTION
**Pull request by voidale**
I can't use the package without changing it to `url: 's3-eu-west-1.amazonaws.com',` so it will be great if this can be merged

### CODE REVIEW

**Comment by benjreinhart** [comment]
Cool, thank you!

Hmm, it'd be great if we could use the `region` option, since it appears to sometimes (always?) be the same as the subdomain you need. That being said, it seems I can't resolve `us-east-1.amazonaws.com` or `s3-us-east-1.amazonaws.com`, so guessing that won't always work?

However, if using the `region` option is not going to work, it would be nice if we could rename the `url` to something less vague, maybe `awsUrl` or `regionUrl`? Thoughts?

**Comment by voidale** [comment]
@benjreinhart, region and the endpoint are not the same so we can't really use it, i changed it to awsURL if thats better? it's actually not really a region url it's just the aws s3 url images are also accessed from it. so I think it makes sense?

**Comment by benjreinhart** [comment]
👍

File: src/RNS3.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ export class RNS3 {
       contentType: file.type
     });
 
-    let url = `https://${ options.bucket }.s3.amazonaws.com`;
+    let url = `https://${ options.bucket }.${options.url || 's3.amazonaws.com'}`;
```
Comment (benjreinhart) [inline src/RNS3.js]: This needs to be `awsURL`, no?

Also, I go back and forth all the time on whether I think something like url should be all caps or just the u when it comes after something else in a variable name.

`awsUrl` vs `awsURL`

I think I'm leaning towards `awsUrl` but honestly don't care that much.

**Comment by benjreinhart** [comment]
I'll pull this into my app in a bit, test it, then deploy to NPM if it's all good.

**Pull request merged**